packages/@aws-cdk-testing/framework-integ/test/aws-lambda-destinations/test/integ.destinations.ts

```diff
@@ -54,6 +55,22 @@ class TestStack extends Stack {
       onSuccess: new destinations.LambdaDestination(onSuccessLambda),
     });
 
+    const successBucket = new s3.Bucket(this, 'OnSuccessBucket');
+    const failureBucket = new s3.Bucket(this, 'OnFailureBucket');
+
+    new lambda.Function(this, 'S3', {
+      runtime: STANDARD_NODEJS_RUNTIME,
+      handler: 'index.handler',
+      code: lambda.Code.fromInline(`exports.handler = async (event) => {
+        if (event.status === 'OK') return 'success';
+        throw new Error('failure');
+      };`),
+      onFailure: new destinations.S3Destination(failureBucket),
+      onSuccess: new destinations.S3Destination(successBucket),
+      maxEventAge: Duration.hours(4),
+      retryAttempts: 2,
+    });
```

Comment:
I haven't been able to run the tests successfully yet. Tomorrow, the service deployment will reach DUB, and I'll run the test there. Then I'll update the commit message and the integration test snapshot. 
Please review the other changes until then.

Reply:
Updated the test snapshot in a new commit